metagraph/modules/shared_data/src/main/scala/com/my/dor_metagraph/shared_data/types/Types.scala

```diff
@@ -5,49 +5,111 @@ import derevo.circe.magnolia.{decoder, encoder}
 import derevo.derive
 import org.tessellation.currency.dataApplication.{DataCalculatedState, DataOnChainState, DataUpdate}
 import org.tessellation.schema.address.Address
+import org.tessellation.schema.balance.Balance
+import org.tessellation.schema.transaction.RewardTransaction
 
 object Types {
-  val EPOCH_PROGRESS_1_DAY: Long = 60 * 24
+  val EpochProgress1Day: Long = 60 * 24
 
   //Friday, 1 September 2023 00:00:00
-  val MINIMUM_CHECK_IN_TIMESTAMP = 1693526400L
+  val MinimumCheckInTimestamp: Long = 1693526400L
```

Comment:
```suggestion
  val MinimumCheckInSeconds: Long =  java.time.Instant.parse("2023-09-01T00:00:00.00Z").toEpochMilli / 1000L
```
Clearer? It also removes the need for the comment. FYI,
```
scala> java.time.Instant.parse("2023-09-01T00:00:00.00Z").toEpochMilli / 1000L
val res2: Long = 1693526400
```